docs/chapter4-payload-handoff-format.rst

```diff
@@ -35,3 +35,139 @@ architecture, either 32 or 64 bits.
 Many of these sections reference the Devicetree Specification [DTspec]_.
 
 TBD (Nodes)
+
+4.2.8 Node: /memory (R)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The memory node is required to describe the physical-memory layout for the system. If a system has multiple ranges of memory, multiple memory nodes can be created, or the ranges can be specified in the reg property of a single memory node. See the Devicetree Specification ("/memory node" section) for details.
+
+.. tabularcolumns:: | p{3cm} p{0.75cm} p{2cm} p{9.5cm} |
+.. table:: Node: memory@base_address
+
+  ==================== ========= =============== ======================================================
+   Property Name        Usage     Value Type      Definition
+  ==================== ========= =============== ======================================================
+   device_type           R       string          Value shall be �memory�
```

Comment:
What is the strange character around 'memory'?

Perhaps use `'memory'` ?

Reply:
sure. it was fixed.